android/local.properties

```diff
@@ -1,5 +1,5 @@
-sdk.dir=C:\\Users\\Srivatsa\\AppData\\Local\\Android\\sdk
-flutter.sdk=D:\\flutter
+sdk.dir=/Users/rajkharvar/Library/Android/sdk
```

Comment:
Should this file be in gitignore?

Reply:
Yes, it needs to be in .gitignore file and i have already added it. I don't know why it has been added in the staging area. Anyway will fix it while committing for a splash screen.

Reply:
Ok. Do you want to do the splash screen in same pr? Or shall I merge this one?

Reply:
At this instance please merge it. I will be sending another pr in the future if splash screen for both android and ios devices is completed successfully.